test/linodes/backups.bats

```diff
@@ -0,0 +1,105 @@
+load '../test_helper/bats-support/load'
+load '../test_helper/bats-assert/load'
+load '../common'
+
+# ##################################################################
+# #  WARNING: THIS TEST WILL DELETE ALL OF YOUR LINODES            #
+# #  WARNING: USE A SEPARATE TEST ACCOUNT WHEN RUNNING THESE TESTS #
+# ##################################################################
+
+setup() {
+    export timestamp=$(date +%s)
+    clean_linodes="false"
+    linode_id=$(linode-cli linodes list --format id --text --no-header | head -n 1)
+}
+
+teardown() {
+    unset timestamp
+
+    if [ "$clean_linodes" = "true" ];
+    then
+        run removeLinodes
+    fi
+}
+
+@test "it should create a linode with backups disabled" {
+    run createLinode
+    local linode_id=$(linode-cli linodes list --format id --text --no-header | head -n 1)
+    run linode-cli linodes backups-cancel $linode_id --text
+
+    assert_failure
+    assert_output --partial "Request failed: 400"
+    assert_output --partial "This Linode does not have a backup service to cancel."
+}
+
+@test "it should enable backups" {
+    run linode-cli linodes backups-enable $linode_id \
+        --text \
+        --no-headers
+    assert_success
+
+    run linode-cli linodes backups-cancel $linode_id \
+        --text \
+        --no-headers
+    assert_success
```

Comment:
Again, you could just check if backups are enabled

Reply:
👍